Mathlib/RingTheory/Ideal/Pointwise.lean

```diff
@@ -0,0 +1,115 @@
+/-
+Copyright (c) 2024 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser
+-/
+import Mathlib.RingTheory.Ideal.Maps
+
+
+/-! # Pointwise instances on `Ideal`s
+
+This file provides the action `Ideal.pointwiseMulAction` which morally matches the action of
+`mulActionSet` (though here an extra `Ideal.span` is inserted).
+
+This actions is available in the `Pointwise` locale.
+
+## Implementation notes
+
+This file is similar (but not identical) to `RingTheory/Subring/Pointwise.lean`.
+Where possible, try to keep them in sync.
+
+-/
+
+
+open Set
+
+variable {M R : Type*}
+
+namespace Ideal
+
+section Monoid
+
+variable [Monoid M] [CommRing R] [MulSemiringAction M R]
+
+/-- The action on an ideal corresponding to applying the action to every element.
+
+This is available as an instance in the `Pointwise` locale. -/
+protected def pointwiseMulSemiringAction : MulSemiringAction M (Ideal R) where
+  smul a I := Ideal.map (MulSemiringAction.toRingHom _ _ a) I
+  one_smul I :=
+    congr_arg (I.map ·) (RingHom.ext <| one_smul M) |>.trans I.map_id
+  mul_smul _a₁ _a₂ I :=
```

Comment:
Is there a reason you used `_a₁` and not just `_`?

Reply:
Yes, to make the goal view easier to understand. It seems the linter no longer requires the `_`s though.